security/yubi/windows/setup-yubi-windows.ps1

```diff
@@ -0,0 +1,91 @@
+# if you are using network drives (e.g. for this repo), make sure that they are linkable between elevated and non-elevated sessions 
+# If so, please set registry entry: "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System" (simply Win+R -> regedit -> Enter)
+# Value "EnableLinkedConnections" as DWORD32 -> Value: 1
+
+# 
+# ELEVATE SHELL
+#
+#some things here need admin permission. so we directly go to admin mode
+if (-not ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator))
+{
+
+    $answer = Read-Host "Setup needs an elevated powershell to write the host file. Do you want to elevate the powershell now? [Y/n]"
+    if ($answer.ToLowerInvariant() -eq "n") {
+        Write-Host -ForegroundColor Red "Closing due to user abortion... (phrasing sounds off)"
+        exit
+    }
+    else {
+        # Get the script path
+        $scriptPath = $MyInvocation.MyCommand.Definition
+
+        # Create a new PowerShell process with administrator privileges
+        $newProcess = New-Object System.Diagnostics.ProcessStartInfo "powershell"
+        $newProcess.Arguments = "-NoProfile -NoExit -Command `"cd '$(Split-Path -Path $scriptPath -Parent)' ; & '$scriptPath'`""
+        $newProcess.Verb = "runas"
+        [System.Diagnostics.Process]::Start($newProcess)
+        exit
+    }
+}
+
+# TODO:
+# install choco itself
+
+# Install Dependencies using choco
+choco install -y gpg4win
+
+
+# Basic GPG Setup
+Copy-Item -Path "gpg-agent.conf" -Destination "$HOME\AppData\Local\gnupg\gpg-agent.conf" -Force -Confirm:$false
```

Comment:
There is also an environment variable for the local appdata path:

$env:LOCALAPPDATA